controlplane/kubeadm/internal/workload_cluster_etcd_test.go

```diff
@@ -32,96 +32,9 @@ import (
 	"sigs.k8s.io/cluster-api/controlplane/kubeadm/internal/etcd"
 	fake2 "sigs.k8s.io/cluster-api/controlplane/kubeadm/internal/etcd/fake"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
-func TestUpdateEtcdVersionInKubeadmConfigMap(t *testing.T) {
```

Comment:
If I'm not wrong this test should be migrated to kubeadm_config_map_test.go

Reply:
The test method `TestUpdateETCDMeta()` in the file `kubeadm_config_map_test.go` tests the same functionality.

Reply:
ok, thanks for clarifying this